swaybar/render.c

```diff
@@ -706,26 +754,35 @@ void render_frame(struct swaybar_output *output) {
 
 	free_hotspots(&output->hotspots);
 
+	struct render_call ctx;
```

Comment:
Nit: `ctx = { 0 }` so we don't get garbage pointers in coredumps in case we forget to initialize a field.